attributes/default.rb

```diff
@@ -31,3 +31,6 @@
 default['activemq']['enable_stomp'] = true
 default['activemq']['use_default_config'] = false
 default['activemq']['install_java'] = true
+# set to 'transport.enabledProtocols=TLSv1,TLSv1.1,TLSv1.2'
+# to disable sslv3 and protect against poodle
+default['activemq']['transport_protocols'] = nil
```

Comment:
The template has
```erb
<transportConnector name="openwire" uri="tcp://0.0.0.0:61616?<%= 
  node['activemq']['transport_protocols'] ? node['activemq']['transport_protocols'] + '&amp;' : '' 
%> . . .
```
so it does not insert `transport.enabledProtocols=` automatically.  I don't think that string should be included in the node property, it should be added automatically if the node property is set:

```erb
<transportConnector name="openwire" uri="tcp://0.0.0.0:61616?<%= 
  node['activemq']['transport_protocols'] ? 'transport.enabledProtocols=' + node['activemq']['transport_protocols'] + '&amp;' : '' 
%> . . .
```